### DISCUSSION TITLE
Misbehaving over SSH

### DISCUSSION THREAD

**Discussion by qwelias** [ghostty-org/ghostty#3650] [Feature Requests, Ideas] [answered]
I've noticed a number of issues when using an SSH session with ghostty.

1. `ctrl+l` does not clear the screen, it just makes a new prompt
![image](https://github.com/user-attachments/assets/73bb611a-9c3e-4f36-b8a7-b3d83c36cb7b)
here the first group of prompts are caused by `ctrl+l` -- they have no empty lines in-between
the following group is caused by regular `enter` for comparison, spaced as expected

2. all keys duplicate and completely break the prompt:
![image](https://github.com/user-attachments/assets/3b46654b-e184-4910-a121-dc1c182daba8)
the result of pressing `a` 4 times, followed by 3 `backspace`s

![image](https://github.com/user-attachments/assets/5ade8a8f-fb85-4e52-ac1f-365263b85f25)
the result of pressing `s` 4 times, followed by 3 `backspace`s

![image](https://github.com/user-attachments/assets/1e9c9c11-46ad-479a-8b67-6c0f84ca948c)
the result of pressing `f` 4 times, followed by 3 `backspace`s

Notes:
I'm using zsh.
If I switch to bash in the same SSH session then problem (1) remains, but problem (2) resolves.

**Comment by rhodes-b** [comment, marked as answer]
Please read this
https://ghostty.org/docs/help/terminfo
